tests/test_browser.go

```diff
@@ -187,6 +195,13 @@ func (b *testBrowser) staticURL(path string) string {
 	return b.URL("/" + testBrowserStaticDir + "/" + path)
 }
 
+// WsURL returns the Websocket URL that the test browser is listening on for CDP clients.
+func (b *testBrowser) WsURL() string {
+	b.t.Helper()
```

Comment:
Do you think making this a helper necessary? I believe we can access its field in tests instead of via an exported method. WDYT?

Reply:
Agree, this is unnecessary and does not make sense for a direct getter. Changing it :+1:

Reply:
Changed, see [here](https://github.com/grafana/xk6-browser/blob/c421358a0be874c99dc930ddddefaaa1ee7e12f5/tests/browser_type_test.go#L18).